packages/website/src/pages/index/group/components/GroupLayout.tsx

```diff
@@ -30,14 +31,15 @@ const GroupTabsItems = [{
 }]
 const groupTabsByKey = keyBy(GroupTabsItems, 'key')
 
-type IGroupLayoutProps = PropsWithChildren<{group: GroupProfile|undefined, curTab: GroupTabs, groupName: string}>
+type IGroupLayoutProps = PropsWithChildren<{ group: GroupProfile | undefined, curTab: GroupTabs, groupName: string }>
 
 const GroupLayout: React.FC<IGroupLayoutProps> = ({ group, children, curTab, groupName }) => {
-  const navigate = useNavigate()
+  const [, navigate] = useTransitionNavigate()
   const handleTabChange = (key: string) => navigate(groupTabsByKey[key as GroupTabs].to(groupName))
   return (
     <div className={styles.pageContainer}>
-      <GroupHeader group={group} />
+      {/* TODO:@waua error boundary */}
```

Comment:
```suggestion
      {/* TODO: @waua error boundary */}
```

空个空格吧，有编辑器对 TODO 有高亮支持